src/components/Clock/Clock.tsx

```diff
@@ -0,0 +1,49 @@
+import { Component } from 'react';
+
+function getFormattedDate(): string {
+  const date = new Date();
+
+  return date.toUTCString().slice(-12, -4);
+}
+
+interface State {
+  today: string;
+}
+
+interface Props {
+  clockName: string;
+}
+
+export class Clock extends Component<Props> {
+  state: Readonly<State> = {
+    today: getFormattedDate(),
+  };
+
+  dateTimerId = window.setInterval(() => {
+    const date = getFormattedDate();
+
+    this.setState({ today: date });
+
+    // eslint-disable-next-line no-console
+    console.info(date);
+  }, 1000);
```

Comment:
- `dateTimerId` should be a nullable class member
- we should set interval when mounting